addons/html_builder/static/src/builder/components/option_container.js

```diff
@@ -22,6 +26,37 @@ export class OptionsContainer extends Component {
     }
 
     get title() {
-        return this.env.getEditingElement().dataset.name;
+        const editingElement = this.env.getEditingElement();
```

Comment:
Funny, i have a commit for this https://github.com/odoo-dev/odoo/pull/3804/commits/7e563b8e7c5e22f95772f2a7170a26d04fb96cf1

Reply:
Ah I didn't know there was already an utils :open_mouth: I will use it instead and I will surely move the utils file in the `utils` folder :+1: